ui/shared/utils/constants.js

```diff
@@ -85,6 +86,28 @@ export const FAMILY_ANALYSIS_STATUS_OPTIONS = [
   { value: FAMILY_STATUS_WAITING_FOR_DATA, color: '#FFC107', name: 'Waiting for data' },
 ]
 
+// SUCCESS STORY
+
+export const FAMILY_SUCCESS_STORY_NOVEL_DISCOVERY = 'N'
+export const FAMILY_SUCCESS_STORY_ALTERED_CLINICAL_OUTCOME = 'A'
+export const FAMILY_SUCCESS_STORY_COLLABORATION = 'C'
+export const FAMILY_SUCCESS_STORY_TECHNICAL_WIN = 'T'
+export const FAMILY_SUCCESS_STORY_DATA_SHARING = 'D'
+export const FAMILY_SUCCESS_STORY_OTHER = 'O'
```

Comment:
in general, you should only export constants if they are used outside of the file, which I don't think these are